### DISCUSSION TITLE
Line Break and Multi-Track Export

### DISCUSSION THREAD

**Discussion by padlup0037** [helge17/tuxguitar#183] [General]
Hello and congratulations on your work!
I would like to create a PDF document with blues licks instead of a complete piece. Is there a way to have the partitions appear one below the other or have line breaks?
 I tried creating a track for each lick, and on the screen, it looks great. However, when exporting to PDF, each lick is on a separate page
. Is there an existing solution for this? 
If not, could this be considered for a future version? Perhaps adding a checkbox for "all tracks on the same PDF" during export?

Another point, during PDF export, the releases of bends are not included. 
![image](https://github.com/helge17/tuxguitar/assets/155154334/e4016069-a42f-4276-bd99-7129a024c13d)

[licks.pdf](https://github.com/helge17/tuxguitar/files/13793136/licks.pdf)

**Comment by guiv42** [comment]
Hi
as described in [Contribute doc](https://github.com/helge17/tuxguitar/blob/master/doc/contribute.md) (see "Suggest improvements" section), we use "issues" and not "discussions" to submit new feature requests.
So I moved this discussion to #202
